ignite/handlers/time_limit.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import time
+from typing import Optional
+
+from ignite.engine import Engine
+
+log = logging.getLogger(__name__)
+
+__all__ = ["TimeLimit"]
+
+
+class TimeLimit:
+    """TimeLimit handler can be used to control training time for computing environments where session time is limited.
+
+    Args:
+        limit_sec (int, optional): Maximum time before training terminates (in seconds). Defaults to 3600.
+
+    Examples:
+
+        .. code-block:: python
+
+            from ignite.engine import Events
+            from ignite.handlers import TimeLimit
+            handler = TimeLimit(288800) # 8 hours of training
+            trainer.add_event_handler(Events.ITERATION_COMPLETED, handler)
+    """
+
+    def __init__(self, limit_sec: Optional[int] = 3600):
```

Comment:
```suggestion
    def __init__(self, limit_sec: Optional[int] = 288800):
```
Let's say 8h by default ?